lib/api/metrics.js

```diff
@@ -0,0 +1,11 @@
+export const sendMetricsReport = ({ Log, ID, SomeJSON }) => ({
+    method: 'post',
+    url: 'metrics',
+    data: { Log, ID, SomeJSON }
+});
+
+export const sendSimpleMetrics = ({ Category, Action, Label }) => ({
+    method: 'get',
+    url: 'metrics',
+    data: { Category, Action, Label }
```

Comment:
⚠️ it should be `params` since it's a `get` request.
```suggestion
    params: { Category, Action, Label }
```

Reply:
Yep, it's a get request, but `{ Category, Action, Label }` go in the body of the request. I thought `params` was meant to be used in the `url` only, while `data` was for the body. So this logic is wrong?

Reply:
Seems to be a mistake in the API spec, I let you confirm with @bartbutler please.

Reply:
Yep, he says there's a mistake. I'm opening an issue about that. I guess the mistake is simply corrected by changing the api call from GET to POST (I guess it was meant to be POST all along), so I'll adapt the `metrics.js` file correspondingly.